Userland/Applications/PixelPaint/LevelsDialog.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2022, Torsten Engelmann
```

Comment:
Please provide an email as well, usually people use the same one as they do for their git commits, or a GitHub noreply email in case you don't want it in the source code for whatever reason (it's still in the commit metadata ofc)